src/CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ set(SOURCES
     ports.c
     recv.c
     send.c
+#    send-linux.c
```

Comment:
It looks like there's some cleanup needed here.

Reply:
Good point, I've removed this line and moved it to only being added if compiling on a non-Apple, non-BSD system.